cudl-sandbox/terraform.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.44.0"
+    }
+    local = {
+      source  = "hashicorp/local"
+      version = "~> 2.1.0"
+    }
+  }
+
+  backend "s3" {
+    bucket         = "sandbox-cudl-terraform-state"
+    key            = "sandbox-cudl-infra.tfstate"
+    dynamodb_table = "terraform-state-lock-cudl"
+    region         = "eu-west-1"
+  }
+
+  required_version = "~> 1.7.5"
```

Comment:
We could, but tracking the latest version is a bit of a moving target. I've tested with 1.7.5 which works. We could go up to 1.8.2 or higher with the next PR?

Reply:
Sure.  Can we make it "> 1.7.5" then to allow a little more range?

Reply:
Pinning it to 1.7.x is suggesting we're okay with patch versions in the 1.7 range, but it's a more doubtful with higher versions. It's indicative of the version used in the most recent deployment. If we use > 1.7.5 we would allow e.g. 3.0.0 which may produce unexpected results if that version was used to deploy (albeit this may be years in the future)

Reply:
Understood, however at the moment we have ">= 0.14.9" so this would be a significant improvement on that. However if we're happy with requiring this specific version for now that's ok by me.